Cargo.toml

```
@@ -13,7 +13,19 @@ Timeouts for futures.
 """
 
 [dependencies]
+wasm-timer = { version = "0.2.4", optional = true }
+wasm-bindgen-crate = { package = "wasm-bindgen", version = "0.2.55", optional = true }
```

```suggestion
wasm-bindgen = { version = "0.2.55", optional = true }
```

Only this change :arrow_up: is needed. The content of your `[features]` section below will still work.